app/interactors/donation/create.rb

```diff
@@ -0,0 +1,17 @@
+class Donation
+  include Charge
+  include Customer
+  include Token
+
+  class Create
+    include Interactor::Organizer
+
+    organize Token::Generate, Customer::Create, Charge::Create
```

Comment:
What if in parallel universe you were creating records. I used ActiveRecord::Base.transaction do ... block to rollback. In Interactor case - we should raise ActiveRecord::Rollback exception or what ?